.github/workflows/check-copilot-commits.yml

```diff
@@ -0,0 +1,59 @@
+---
+# This workflow validates that pull request commits are not
+# authored by GitHub Copilot. According to Camunda's Copilot
+# policies (https://confluence.camunda.com/x/MoegDg),
+# commits must be properly attributed to human developers
+# to ensure accountability and maintain clean git history. This check
+# helps prevent future history rewrites.
+#
+# The workflow will fail if any commits in the PR are authored by
+# GitHub Copilot, requiring the developer to recreate those commits
+# with proper attribution.
+#
+# owner: @camunda/monorepo-devops-team
+name: Check Copilot Commits
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+jobs:
+  check-copilot-commits:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+    - name: Check for Copilot commits
+      id: copilot-check
+      uses: camunda/infra-global-github-actions/check-copilot-commits@copilot-checker
+      with:
+        repository: ${{ github.repository }}
+        branch: ${{ github.event.pull_request.head.ref }}
+        base-branch: ${{ github.event.pull_request.base.ref }}
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Fail if Copilot commits found
+      if: steps.copilot-check.outputs.copilot-commits-found == 'true'
+      run: |
+        # Create clickable links for each commit hash
+        commit_hashes="${{ steps.copilot-check.outputs.copilot-commits-hashes }}"
+        repo_url="https://github.com/${{ github.repository }}"
+        commit_links=""
+
+        for hash in ${commit_hashes}; do
+          if [ -n "${commit_links}" ]; then
+            commit_links="${commit_links}, "
```

Comment:
I think printing commits as list using newlines instead of comma will improve readability when multiple commits are reported

Reply:
As explained in the sibling pr in infra-core, annotations don't support multiline strings.
But [this example](https://github.com/camunda/infra-core/actions/runs/19102822618/job/54578809055?pr=10852) shows that single line works as well.